server/bundles/io.cloudbeaver.server/schema/service.core.graphqls

```diff
@@ -458,6 +458,22 @@ type LogEntry {
     stackTrace: String
 }
 
+
+type TransactionLogInfos {
```

Comment:
Why types are in 1 gql and function in another? Let's move everything in sql gql.